etc/httpd/global.conf

```diff
@@ -603,7 +603,7 @@ RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?mobile/android-download(-beta)?\.html)$ /
 RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?projects/calendar(/.*)?)$ /b$1 [PT]
 
 # bug 947890
-RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/latest/releasenotes(/?)$ /b/$1firefox/latest/releasenotes$2 [PT]
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/(?:latest/)?releasenotes(/?)$ /b/$1firefox/latest/releasenotes$2 [PT]
```

Comment:
It only returns a 500 if the request actually gets to bedrock, so we get 500s in AWS, but in SCL3 it returns a 404 because it doesn't pass through. Apologies, I meant to add a comment earlier explaining that.

Would you prefer a separate line that passes `/firefox/releasenotes/` through as `/firefox/releasenotes/` and doesn't explicitly link them together in `global.conf`?